integration/kubernetes/init.sh

```diff
@@ -81,14 +78,9 @@ sleep_time=5
 cmd="kubectl get pods --all-namespaces | grep 'coredns.*1/1.*Running'"
 waitForProcess "$dns_wait_time" "$sleep_time" "$cmd"
 
-if [ "${use_runtime_class}" == true ]; then
-	runtimeclass_files_path="${SCRIPT_PATH}/runtimeclass_workloads"
-	echo "Install RuntimeClass resource definition"
-	kubectl apply -f \
-		"https://raw.githubusercontent.com/kubernetes/kubernetes/v${kubernetes_version/-*}/cluster/addons/runtimeclass/runtimeclass_crd.yaml"
-	echo "Create kata RuntimeClass resource"
-	kubectl create -f "${runtimeclass_files_path}/kata-runtimeclass.yaml"
-fi
+runtimeclass_files_path="${SCRIPT_PATH}/runtimeclass_workloads"
+echo "Create kata RuntimeClass resource"
+kubectl create -f "${runtimeclass_files_path}/kata-runtimeclass.yaml"
```

Comment:
This yaml needs to be updated like here https://github.com/clearlinux/cloud-native-setup/blob/master/clr-k8s-examples/8-kata/kata-qemu-runtimeClass.yaml#L2-L6

Reply:
done